src/modules/xplorers-ai/xplorersAIBody.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import XplorersAISearch from "./xplorersAISearch";
+import XplorersAIChatBox from "./xplorersAIChatBox";
+import { useXplorersAI, FileUploadEvent } from "@/hooks/useXplorersAI";
+import { ChangeEvent } from "react";
+
+export interface XplorersAIProps {
+  handleFileUpload: (e: FileUploadEvent) => void;
+  handleSubmit: (e: any) => Promise<void>;
+  handleInputChange: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+}
+const XplorersAIBody = () => {
+  const {
+    handleFileUpload,
+    handleSubmit,
+    handleInputChange,
+    getMessages,
+    value,
+  } = useXplorersAI();
```

Comment:
If we are just passing these destructured values as a prop, why do we need to call this hook here ? why not directly inside the `XplorersAISearch` component !